src/main/java/website/frontrow/util/FileNameCollector.java

```diff
@@ -0,0 +1,53 @@
+package website.frontrow.util;
+
+import website.frontrow.logger.Log;
+import website.frontrow.logger.Logable;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/**
+ * A method for indexing the levels in the resources and returning them.
+ */
+public class FileNameCollector
+    implements Logable
+{
+    /**
+     * This method is used for obtaining the files names in a directory.
+     * @param dir Input a string of the directory, levels are in: /level/.
+     * @return Returns a String array with the directory + their file names.
+     * @throws URISyntaxException Throws URISyntaxException when URI is not valid.
+     */
+    public String[] obtain(String dir) throws URISyntaxException
```

Comment:
Very nice, but maybe do it statically, since you don't use any kind of objects. Entirely up to you though.